lib/core/test/unit/Cardano/Wallet/DB/Arbitrary.hs

```diff
@@ -506,7 +507,11 @@ instance Eq XPrv where
 
 instance Arbitrary (Hash purpose) where
     arbitrary = do
-        Hash . convertToBase Base16 . BS.pack <$> vectorOf 32 arbitrary
+        Hash . convertToBase Base16 . BS.pack <$> vectorOf 16 arbitrary
```

Comment:
Generate only `16` initial bytes here since the conversion to `Base16` will double the underlying size. It doesn't matter much but it gives nicer QC failures with shorter ids.